src/panels/logbook/ha-logbook.ts

```diff
@@ -284,20 +279,13 @@ export class HaLogbook extends LitElement {
   private async _subscribeLogbookPeriod(
     logbookPeriod: LogbookTimePeriod
   ): Promise<void> {
-    if (this._subscribed) {
+    if (this._unsubLogbook) {
       return;
     }
     try {
-      this._subscribed = await subscribeLogbook(
+      this._unsubLogbook = subscribeLogbook(
```

Comment:
As this is also no awaited, it will not throw. To fix that you could do:

```ts
try {
	this._unsubLogbook = subscribeLogbook(...);
	await this._unsubLogbook;
} catch (e) {
	...
}